tests/scripts/pulpcore/test_task.sh

```diff
@@ -41,9 +44,16 @@ task=$(echo "$ERROUTPUT" | grep -E -o "${PULP_API_ROOT}api/v3/tasks/[-[:xdigit:]
 task_uuid="${task%/}"
 task_uuid="${task_uuid##*/}"
 expect_succ pulp task show --wait --uuid "$task_uuid"
+reserved_resource="$(echo "$OUTPUT" | jq -r '.reserved_resources_record[0]')"
+created_resource="$(echo "$OUTPUT" | jq -r '.created_resources[0]')"
 expect_succ test "$(echo "$OUTPUT" | jq -r '.state')" = "completed"
 
 expect_succ pulp task list --name-contains file
+expect_succ pulp task list --parent-task "$task" --worker "$worker"
+expect_succ pulp task list --parent-task "$task_uuid" --worker "$worker_name"
+expect_succ pulp task list --started-before "2021-12-01" --started-after "2022-06-01"
+expect_succ pulp task list --finished-before "2021-12-01" --finished-after "2022-06-01"
```

Comment:
Can we add a test that shows the full H/M/S date format working, as well as a negative one failing when you use, say, a US-formatted date?

Reply:
We can easily support what the server accepts.

Reply:
That's actually not true. Click interprets the date for us.

Reply:
One benefit of the tests is an examples of "how to use this", so if a param can have two formats, I'm a fan of showing both in the tests.